### ISSUE TITLE
Lab 10 Run Through

### ISSUE THREAD

**Issue by scollier** [openshift/federation-dev#156]
I did have an issue with the argocd app not deleting.

```
$ argocd app list
NAME                CLUSTER                                                               NAMESPACE     PROJECT  STATUS     HEALTH   SYNCPOLICY  CONDITIONS     REPO                                                                                             PATH                                                TARGET
cluster1-kustomize  https://api.cluster-brno-32bd.brno-32bd.sandbox1682.opentlc.com:6443  dev-web-site  default  Synced     Healthy  Auto        <none>         http://gogs.apps.cluster-brno-32bd.brno-32bd.sandbox1682.opentlc.com/student/federation-dev.git  labs/lab-5-assets/base                              master
cluster1-mongo      https://api.cluster-brno-32bd.brno-32bd.sandbox1682.opentlc.com:6443  mongo         default  OutOfSync  Missing  Auto        DeletionError  http://gogs.apps.cluster-brno-32bd.brno-32bd.sandbox1682.opentlc.com/student/federation-dev.git  labs/lab-6-assets/overlays/cluster1                 master

```

**Comment by mvazquezc** [comment]
If we hit this issue again we should remove the finalizer from the ArgoCD Application CR. I'll keep this open while we try to reproduce the error.

**Comment by mvazquezc** [comment]
Fixed by #162
